consoles/node-cpu.html

```diff
@@ -1,11 +1,13 @@
 {{ template "head" . }}
 
 {{ template "prom_right_table_head" }}
-  <tr><th colspan="2">CPU</th></tr>
+  <tr>
+    <th colspan="2">CPU(s): {{ template "prom_query_drilldown" (args (printf "scalar(count(count by (cpu)(node_cpu{job='node',instance='%s'})))" .Params.instance)) }}</th>
```

Comment:
I did that initially, but i didn't really like that it was the only non-percentage in the table. But that's pretty minor, so i'm happy to revert to that.